bundles/amazon/ack/elasticache/README.md

```diff
@@ -0,0 +1,58 @@
+# AWS Controllers for Kubernetes - Elasticache Redis Instance
+
+Status: Experimental
+
+## Use
+
+Create a `CacheSubnetGroup`, for example:
```

Comment:
**question** Could/should we create this as part of the package install?

Reply:
We _could_, but that would mean providing even more ephemeral values to the package. We're currently accepting a list of Security Group IDs, which are pretty much ephemeral (you destroy and recreate the SGs and you get different IDs, meaning that you'd need to update your installed package), but there's no other way to do it.
If we wanted to create the `CacheSubnetGroup` as part of the package then we'd need to have also the IDs of the subnets to create the group for. Unfortunately there's no such a thing as data sources for Terraform I could use to read the IDs (i.e. matching a tag).
However, we could have also a boolean like `createCacheSubnetGroup` as input value (along with optional subnet IDs) to determine if the user wants the `CacheSubnetGroup` to be created. We just need some more ytt logic but it's doable.

Reply:
We also didn't include `dbSubnetGroup` as part of the RDS bundle for similar reasons. Whatever we decide we should ensure we do the same for the RDS bundle too.

Reply:
Cool makes sense - perhaps lets leave it as it is for now then.